tech_leads/README.md

```diff
@@ -0,0 +1,98 @@
+# Tech Leads
+
+## Current roster
+
+* Backend - [CM Lubinski](https://gsa-tts.slack.com/messages/@cm)
+* Data - [Tony Garvan](https://gsa-tts.slack.com/messages/@tony)
+* DevOps - [Adam Kendall](https://gsa-tts.slack.com/messages/@adamkendall)
+* Front-End - [Marco Segreto](https://gsa-tts.slack.com/messages/@marco)
+* Security - [DKP](https://gsa-tts.slack.com/messages/@dkp)
+
+## Responsibilities
+
+Technical leads are selected to improve the technical quality of our projects. 
+These qualities cover high-level topics, such as system architectures and 
+deployment strategies, frequent tasks like library selection and developer 
+work-flow, down to line-level style guides. Best practices may arise from each 
+lead's experience, but more often come from aggregating knowledge across 18F 
+projects. Leads need to maintain a good sense of what problems are recurring 
+across teams and be prepared to ensure solutions propagate.
+
+The leads also serve as point people around engineering-wide issues that arise 
+within their domains. These include triaging security issues, ensuring the 
+maintenance of shared libraries, pinch-hitting when teams are missing specific 
+skills, and the like. Leads therefore need to be generally available to 
```

Comment:
"Technical leads may also serve as subject matter aspects for business development and staffing."